### PR TITLE
Update for 0.1 with pointer to current docs

### PR DESCRIPTION
**Pull request by ldemailly**
Mirror of #406

### CODE REVIEW

**Comment by ldemailly** [comment]
build istio.github.io/presubmit

File: _docs/index.md [hunk 1]
```diff
@@ -10,9 +10,11 @@ type: markdown
 ---
 {% include home.html %}
 
-# Welcome
```
Comment (geeknoid) [inline _docs/index.md]: Can we leave the title as Welcome? This matches the tab name, and matches what we do in all other sections like Concepts, Tasks, Reference. Maybe just:

# Welcome - Istio 0.1

Comment (ldemailly) [inline _docs/index.md]: I changed it to match what you asked in #406 and to make it extra clear this set is potentially put of date.

Comment (ldemailly) [inline _docs/index.md]: changed - PTAL

**Pull request merged**